components/sections/Sponsors.tsx

```diff
@@ -119,9 +151,48 @@ export default function Sponsors(): ReactElement {
               minWidth="72px"
             />
           </Grid>
-          <Headline marginTop="0.6in" marginBottom="1in">
-            Stay tuned for more!
-          </Headline>
+          <Grid
+            width="100%"
+            gridTemplateColumns={[
+              "repeat(3, 1fr)",
+              "repeat(3, 1fr)",
+              "repeat(5, 1fr)",
+            ]}
+            gridTemplateAreas={[thinSchoolGrid, thinSchoolGrid, wideSchoolGrid]}
+            justifyItems="center"
+            alignItems="center"
+          >
+            <SponsorImage
+              src={`${imgPath}/universities/calgary.svg`}
+              gridArea="calgary"
+              width={["60%", "60%", "25%"]}
+              minWidth="72px"
+            />
+            <SponsorImage
+              src={`${imgPath}/universities/carleton.svg`}
+              gridArea="carleton"
+              width={["60%", "60%", "25%"]}
+              minWidth="72px"
+            />
+            <SponsorImage
+              src={`${imgPath}/universities/toronto.svg`}
+              gridArea="toronto"
+              width={["60%", "60%", "50%"]}
```

Comment:
Hmm I'm just noticing this now but on the last breakpoint, should this (and waterloo's) be set to "25%" like the rest?

Reply:
Thanks for catching that! I just pushed a fix